lib/hooks/useUserRolesById/useUserRolesById.js

```diff
@@ -0,0 +1,49 @@
+import { useQuery } from 'react-query';
+
+import { useNamespace, useOkapiKy } from '@folio/stripes/core';
+
+import { ROLES_USERS_API, USER_ROLES_LIMIT } from '../../constants';
+
+/* Get list of user-roles by user IDs. That is, a list of roles assigned to the requested user IDs.
+// The response shape is:
+{
+  "userRoles": [
+    {
+        "userId": "<USER_ID>",
+        "roleId": "<ROLE_ID>",
+        "metadata": {
+            "createdDate": "<UTC_DATETIME>",
+            "createdBy": "<USER_ID>"
+        }
+    },
+    ...
+  ]
+}
+*/
+
+export function useUserRolesById(userId, options = {}) {
+  const [namespace] = useNamespace({ key: 'user-roles' });
+  const { tenantId } = options;
+  const ky = useOkapiKy({ tenant: tenantId });
+
+  const queryFn = async () => {
+    const response = await ky.get(`${ROLES_USERS_API}/${userId}`, {
+      searchParams: {
+        limit: USER_ROLES_LIMIT,
+      },
+    }).json();
+    return response.userRoles;
+  };
+
+  const { data, isLoading, isFetching } = useQuery(
+    [namespace, 'roles/users', userId, tenantId],
+    queryFn,
+    {
+      enabled: Boolean(tenantId && userId),
```

Comment:
I don't think `tenantId` has to be a reason to disable the hook. The `tenatId` can be omitted and `useOkapiKy` will use the system tenant. In the non-ECS env we don't need to specify `tenantId` explicitly, so it should not block the query.

Besides, it also makes sense to extract `enabled` option and combine with your condition:
```ts
const {
  enabled = true,
  tenantId,
} = options;

// ---

  ...
  enabled: Boolean(enabled && userId),
  ...
```

Reply:
Does the consortium manager work on non-esc envs ?